dietpi/dietpi-config

```diff
@@ -2418,7 +2418,7 @@
 		#Restart Networking
 		G_DIETPI-NOTIFY 2 'Restarting network, please wait...'
 		systemctl daemon-reload
-		#systemctl restart networking #: Does exactly the same as ifdown/up that we already do.
+		systemctl restart networking #: Required to apply static IP changes...
```

Comment:
@Fourdee 
Are you sure that this is required? Since networking service does the same:
`ExecStart=/sbin/ifup -a --read-environment`
- And this does not even include eth0/wlan0 interfaces, if added via `allow-hotplug`, but only `auto` interfaces.
> -a, --all               process all interfaces marked "auto"

On Jessie hotplug interfaces are included (init.d service):
`if ifup -a --exclude=lo $exclusions $verbose && ifup_hotplug $exclusions $verbose`

Reply:
@MichaIng 

Yep got it wrong, this is required to flush DHCP IP's when changing to static:
```
		# ip addr flush dev eth$ETH_DEV_INDEX &> /dev/null
		# ip addr flush dev wlan$WIFI_DEV_INDEX &> /dev/null

```

Reply:
@Fourdee 
Hmm never found an issue when switching to STATIC or changing static IP. Of course this drops active SSH session.
ifdown and ifup reapplies `/etc/network/interface` settings, I thought, including DHCP address request, since `dhclient` is triggered via `if-up.d`.

Where did an interface restart not apply new IP?

Reply:
@MichaIng 

ASUS TB. When connected to SSH:
- Change from DHCP to static
- IP address is still the same, unless ```ip addr flush dev eth0``` was used, or, reboot